### PR TITLE
Analytics

### PR DESCRIPTION
**Pull request by hexxorleiva**
Added the latest Google Analytics syntax with a inclusion so that all subdomains of cravetickets.com are tracked (including beta.cravetickets.com).

### CODE REVIEW

File: templates/html5/html5.php [hunk 1]
```diff
@@ -107,11 +107,19 @@
...
-<script>
-    var _gaq=[['_setAccount','<?=$google_analytics_account?>'],['_trackPageview']]; // Change UA-XXXXX-X to be your site's ID
-    (function(d,t){var g=d.createElement(t),s=d.getElementsByTagName(t)[0];g.async=1;
-    g.src=('https:'==location.protocol?'//ssl':'//www')+'.google-analytics.com/ga.js';
-    s.parentNode.insertBefore(g,s)}(document,'script'));
+<script type="text/javascript">
+
+  var _gaq = _gaq || [];
+  _gaq.push(['_setAccount', '<?=$google_analytics_account?>']);
+  _gaq.push(['_setDomainName', 'cravetickets.com']);
```
Comment (will123195) [inline templates/html5/html5.php]: can't have a domain in skyphp!

Comment (will123195) [inline templates/html5/html5.php]: what does `_setDomainName` actually do?

**Comment by hexxorleiva** [comment]
_setDomainName sets the cookie that Google Analytics Tracking Code (GATC) uses to be able to include tracking on sub-domains of the domain that is stated in the _setDomainName. Otherwise the domain (cravetickets.com) is tracked and none of the sub-domains are included because it wouldn't match the document.domain property in the DOM.

https://developers.google.com/analytics/devguides/collection/gajs/methods/gaJSApiDomainDirectory#_gat.GA_Tracker_._setDomainName

But this is my assumption that since I declared `$google_analytics_account` in the config file that the tracking code that is inside `templates/html5/html5.php` that I added will only be for beta.cravetickets.com. I think in over-looking it now that I might be wrong?

**Comment by hexxorleiva** [comment]
Please view with the cravetix analytics branch.

File: templates/html5/html5.php [hunk 2]
```diff
@@ -107,11 +107,25 @@
...
-    s.parentNode.insertBefore(g,s)}(document,'script'));
+<script type="text/javascript">
+
+  var _gaq = _gaq || [];
+  _gaq.push(['_setAccount', '<?=$google_analytics_account?>']);
+<?
+    if ( $google_analytics_domain ) {
+?>
+  _gaq.push(['_setDomainName', '<?=$google_analytics_domain?>']);
+<?
```
Comment (will123195) [inline templates/html5/html5.php:118]: `<?php`

File: templates/html5/html5.php [hunk 3]
```diff
@@ -107,11 +107,25 @@
...
-<script>
-    var _gaq=[['_setAccount','<?=$google_analytics_account?>'],['_trackPageview']]; // Change UA-XXXXX-X to be your site's ID
-    (function(d,t){var g=d.createElement(t),s=d.getElementsByTagName(t)[0];g.async=1;
-    g.src=('https:'==location.protocol?'//ssl':'//www')+'.google-analytics.com/ga.js';
-    s.parentNode.insertBefore(g,s)}(document,'script'));
+<script type="text/javascript">
+
+  var _gaq = _gaq || [];
+  _gaq.push(['_setAccount', '<?=$google_analytics_account?>']);
+<?
```
Comment (will123195) [inline templates/html5/html5.php:114]: `<?php`

**Pull request closed without merging**